src/cbc_sdk/platform/asset_groups.py

```diff
@@ -94,6 +97,104 @@ def _query_implementation(cls, cb, **kwargs):
         """
         return AssetGroupQuery(cls, cb)
 
+    def list_member_ids(self, **kwargs):
+        """
+        Gets a list of all member IDs in the group, optionally constrained by membership type.
+
+        Required Permissions:
+            group-management(READ)
+
+        Args:
+            **kwargs (dict): Keyword arguments as documented below.
+
+        Keyword Args:
+            rows (int): Maximum number of rows to retrieve from the server. The function may return fewer member IDs
+                        if filtering is applied to the output. Default is 20.
+            start (int): Starting row to retrieve from the server; used to implement pagination. Default is 0.
+            filter (str): Can restrict the types of members that are returned by this method.  Values are "ALL"
+                          to return all members, "DYNAMIC" to return only members that were added via the asset group
+                          query, or "MANUAL" to return only manually-added members.  Default is "ALL".
+
+        Returns:
+            list[int]: List of integer member IDs.
+        """
+        filt = kwargs.get("filter", "ALL")
+        if filt not in AssetGroup.VALID_MEMBER_FILTERS:
+            raise ApiError(f"invalid filter value: {filter}")
+        query_params = {}
+        for name in ("rows", "start"):
+            if name in kwargs:
+                query_params[name] = kwargs[name]
+        member_data = self._cb.get_object(self._build_api_request_uri() + "/members", query_params)
+        id_list = []
+        if filt == "ALL":
+            id_list = [int(m["external_member_id"]) for m in member_data["members"]]
```

Comment:
What if I want to know what the membership type is where an asset group has both dynamic and manual memberships? Instead of a list of ids should we return the entire `members` list